terraform/variables.tf

```diff
@@ -32,6 +32,8 @@ variable dockerhub_username {}
 
 variable dockerhub_password {}
 
+variable paas_app_secrets_file { default = "./terraform/app_secrets.yml" }
+
 variable paas_settings__basic_auth__username {} #secrets
 variable paas_settings__basic_auth__password {} #secrets
 variable paas_settings__dttp__client_id {}      #secrets
```

Comment:
can be removed? as they are moved to the secrets file?

Reply:
removed.